Index: cmake-build-debug/Testing/Temporary/LastTest.log
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Start testing: Dec 09 13:53 WET\n----------------------------------------------------------\nEnd testing: Dec 09 13:53 WET\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cmake-build-debug/Testing/Temporary/LastTest.log b/cmake-build-debug/Testing/Temporary/LastTest.log
--- a/cmake-build-debug/Testing/Temporary/LastTest.log	(revision b16c07931d6d298447c1f6f1807ba780c011e18b)
+++ b/cmake-build-debug/Testing/Temporary/LastTest.log	(date 1702131162511)
@@ -1,3 +1,3 @@
-Start testing: Dec 09 13:53 WET
+Start testing: Dec 09 14:12 WET
 ----------------------------------------------------------
-End testing: Dec 09 13:53 WET
+End testing: Dec 09 14:12 WET
Index: cmake-build-debug/CMakeFiles/clion-Debug-log.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/snap/clion/259/bin/cmake/linux/x64/bin/cmake -DCMAKE_BUILD_TYPE=Debug -G \"CodeBlocks - Unix Makefiles\" -S /home/freitas/CLionProjects/AirTravelingManagementSystemG17 -B /home/freitas/CLionProjects/AirTravelingManagementSystemG17/cmake-build-debug\nCMake Deprecation Warning:\n  Support for \"Extra Generators\" like\n\n    CodeBlocks\n\n  is deprecated and will be removed from a future version of CMake.  IDEs may\n  use the cmake-file-api(7) to view CMake-generated project build trees.\n\n\n-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) \nDoxygen needs to be installed to generate the documentation.\n-- Configuring done (0.1s)\n-- Generating done (0.0s)\n-- Build files have been written to: /home/freitas/CLionProjects/AirTravelingManagementSystemG17/cmake-build-debug\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cmake-build-debug/CMakeFiles/clion-Debug-log.txt b/cmake-build-debug/CMakeFiles/clion-Debug-log.txt
--- a/cmake-build-debug/CMakeFiles/clion-Debug-log.txt	(revision b16c07931d6d298447c1f6f1807ba780c011e18b)
+++ b/cmake-build-debug/CMakeFiles/clion-Debug-log.txt	(date 1702131143081)
@@ -10,6 +10,6 @@
 
 -- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
 Doxygen needs to be installed to generate the documentation.
--- Configuring done (0.1s)
+-- Configuring done (0.0s)
 -- Generating done (0.0s)
 -- Build files have been written to: /home/freitas/CLionProjects/AirTravelingManagementSystemG17/cmake-build-debug
